<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DimensionSuffix0_Dialog</class>
 <widget class="QDialog" name="DimensionSuffix0_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>278</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dimension suffix options</string>
  </property>
  <widget class="QPushButton" name="_1">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>248</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="_103">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>84</y>
     <width>361</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="_101">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>6</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Text options</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="_100">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>10</y>
     <width>283</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Use value text options for suffix text options</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_2">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>248</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="_102">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>38</y>
     <width>297</width>
     <height>43</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SuffixLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Suffix</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="_104">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>38</y>
     <width>57</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Suffix</string>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>SuffixLabel</zorder>
  <zorder>_104</zorder>
  <zorder>_1</zorder>
  <zorder>_103</zorder>
  <zorder>_101</zorder>
  <zorder>_100</zorder>
  <zorder>_2</zorder>
  <zorder>_102</zorder>
 </widget>
 <tabstops>
  <tabstop>_102</tabstop>
  <tabstop>_100</tabstop>
  <tabstop>_101</tabstop>
  <tabstop>_1</tabstop>
  <tabstop>_2</tabstop>
  <tabstop>_103</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
